google-maps/ios/Plugin/CapacitorGoogleMapsPlugin.m

```diff
@@ -19,13 +20,6 @@
    CAP_PLUGIN_METHOD(disableClustering, CAPPluginReturnPromise);
    CAP_PLUGIN_METHOD(destroy, CAPPluginReturnPromise);
    CAP_PLUGIN_METHOD(setCamera, CAPPluginReturnPromise);
-   CAP_PLUGIN_METHOD(getMapType, CAPPluginReturnPromise);
```

Comment:
This is a breaking change - these should be kept, perhaps marked as deprecated.

Reply:
https://github.com/ionic-team/capacitor-plugins/pull/1618/commits/ac752524925fd5ecd4f535a6294d3c3ebe31e647#diff-e887271a6e743973df748b0b83f29f5250ad28c088c82e1aacadfbb4ffc3d1b4R460
The method still exists in the map.ts with the same return type and is marked as deprecetd there. It was just removed from the native implementations because its replaced by the new solution.